notebooks/simulator_tutorial.ipynb

```diff
@@ -0,0 +1,627 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Questions : \n",
+    "\n",
+    "- Should I have more detailed explanations on every field in the initialization of states ?\n",
+    "- Do you wanna talk about neighbor maps in jax_md ?\n",
+    "- Do we talk about the physics engine (only way to control the simulation atm) ?\n",
+    "- Do we talk about braintenberg behaviors ? \n",
+    "- Do we talk about a future refactoring ? \n",
+    "- What kind of custom scene do we create ? "
```

Comment:
Maybe just start with reproducing the same one as in web_interface.md, but doing it programatically.
You could first do it using the pre-implemented braitenberg behaviors, then showing how to program your own behavior (e.g. just reimplementing some of the braitenberg ones).
Then, to see if we could show how to add newborn and remove dead agents